src/run.js

```diff
@@ -248,15 +249,13 @@ const processPage = ({
       // Second, goto the page and evaluate it on the 'networkidle2' event.
       // This gives the page a chance to load any <script defer src="...">
       // and even some JS that does XHR requests right after load.
-      response = await page.goto(pageUrl, {
-        waitUntil: ['domcontentloaded', 'networkidle0']
-      })
+      response = await page.goto(pageUrl, { waitUntil })
       if (!response.ok()) {
         return safeReject(
           new Error(`${response.status()} on ${pageUrl} (second time)`)
         )
       }
-      const evalNetworkIdle = await page.evaluate(() => {
+      const evalWithJavascript = await page.evaluate(() => {
```

Comment:
I just wanted to rename the variable now that the `waitUntil` is "more variable".